Fix:

enttbl.c:560:8: error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]

uni2ascii.c:3176:24: error: implicit declaration of function 'putu8' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- enttbl.c.orig	2011-02-18 10:36:27.000000000 -0600
+++ enttbl.c	2021-02-13 23:14:22.000000000 -0600
@@ -20,6 +20,7 @@
 
 #include "config.h"
 #include <stdlib.h>
+#include <string.h>
 #include "unicode.h"
 
 struct ent {
--- uni2ascii.c.orig	2011-05-14 21:03:13.000000000 -0500
+++ uni2ascii.c	2021-02-13 23:16:55.000000000 -0600
@@ -2786,6 +2786,7 @@
   extern int optopt;
   extern UTF32 UTF8in(int,int *,unsigned char **);
 
+  extern void putu8 (unsigned long);
   extern int CountSlots(char *);
   extern void ListFormatArguments(short);
   extern void SetFormat(char *, int *, short *,int *, short *, short *);
